### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by Yoda-Soda**
Please review the overall structure, with the intent of providing Tema enough information to create the daily emails that go out to students. Content can be continued to be tweaks over the next couple of weeks, and we can create issues for those smaller changes.

## What's in this PR

Overhaul the pre-course week to be in a better order, and add much more content as student's flew through it last time

Structure:
- Day 1: Install all software, sign up for accounts and get familiar with mac
- Day 2: Learn git
- Day 3: More git, and learn about the command line
- Day 4: (optional) Some tools to help you study (trello, notion), and play with design software
- Day 5: (optional) Start Pluralsight videos on web development and get started with HTML 

## What's not in this PR

- There are some issues that will be addressed separately https://github.com/Developers-Institute/WDD01-Client-Side-Pre-Course-Setup/issues, as I'm keen to get the overall structure finalised

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,126 @@
...
+
+---
+
+### Setup GitHub Account
+
+[GitHub](https://github.com/) is a hugely popular cloud-hosted version of Git used by millions of software teams. The majority of courseware and challenges are hosted in GitHub code repositories so you will need a GitHub account to access the course materials.
+
+To be able to access the material you will need to setup a GitHub account:
+
+- [Register a GitHub account](https://github.com/join). You can use your developers institute account to start with, and add your personal email at https://github.com/settings/emails if you want to add your own email address:
```
Comment (lwcooper) [inline README.md:77]: This time I also manually invited students to github, so they had an invite email waiting in their student email account. Maybe worth mentioning that too?

File: README.md [hunk 2]
```diff
@@ -1,80 +1,126 @@
...
+
+[Git Example](<(https://vimeo.com/433825571/bc1830fb90)>)
+
+[![Git example](img/git_example_video.jpg)](https://vimeo.com/433825571/bc1830fb90)
+
+### Self-learning Challenge
+
+Work through the [Quick Start tutorial](https://help.github.com/en/github/getting-started-with-github/quickstart) to get familiar with common GitHub tasks.
+
+At this stage you should have the basic understanding of how to `git clone`, `git add`, `git commit`, `git push`.
```
Comment (lwcooper) [inline README.md:93]: I dont think any of these commands are in the quickstart tutorial. 

Some other links

* https://guides.github.com/introduction/git-handbook/ - this contains commands
* https://www.youtube.com/playlist?list=PLg7s6cbtAD15G8lNyoaYDuKZSKyJrgwB- - this channel works through each git command one at a time, pretty good.

File: README.md [hunk 3]
```diff
@@ -1,80 +1,126 @@
...
+
+### Stretch Content
+
+Get to know git and GitHub better:
+
+- [GitHub Guides](https://guides.github.com/)
+- [GitHub Training & Guides YouTube channel](https://www.youtube.com/user/GitHubGuides/featured)
```
Comment (lwcooper) [inline README.md:100]: oh you've got it here

File: README.md [hunk 4]
```diff
@@ -1,80 +1,126 @@
...
+
+`Note:` getting to grips with git is a marathon and not a sprint. So take it easy.
```
Comment (lwcooper) [inline README.md:102]: 👏

File: README.md [hunk 5]
```diff
@@ -1,80 +1,126 @@
...
+
+[GitHub](https://github.com/) is a hugely popular cloud-hosted version of Git used by millions of software teams. The majority of courseware and challenges are hosted in GitHub code repositories so you will need a GitHub account to access the course materials.
+
+To be able to access the material you will need to setup a GitHub account:
+
+- [Register a GitHub account](https://github.com/join). You can use your developers institute account to start with, and add your personal email at https://github.com/settings/emails if you want to add your own email address:
+
+### Learn the basics of Git
+
+Git is one of the core tools that developers use daily. Watch the [Git Basics](https://git-scm.com/videos) introductory video series on the Git version control system.
```
Comment (lwcooper) [inline README.md:81]: Is it worth expanding on the expectations here, i.e we don't want you to know how to rebase commits, just clone, add, commit and push? if you have those sorted thats all we expect

File: README.md [hunk 6]
```diff
@@ -1,80 +1,126 @@
...
+
+Get to know git and GitHub better:
+
+- [GitHub Guides](https://guides.github.com/)
+- [GitHub Training & Guides YouTube channel](https://www.youtube.com/user/GitHubGuides/featured)
+
+`Note:` getting to grips with git is a marathon and not a sprint. So take it easy.
+
+## Day 3
+
```
Comment (lwcooper) [inline README.md:105]: I'm worried about merging this before we have day 3 and 4. 

I would say, git should be day 3 too. Could we move or expand some of the stuff above? maybe 'make some repos, commit some files, see if you can commit files to some of your class mates repos' or something?

Comment (lwcooper) [reply README.md:105]: I will get some thoughts down about day 3 and 4 before we merge

File: README.md (hunk 1 above)
Comment (Yoda-Soda) [reply README.md:77]: Okay I'll add this.

File: README.md (hunk 2 above)
Comment (Yoda-Soda) [reply README.md:93]: This was more of a summary. I think I'll move that to the bottom of the git section.

File: README.md (hunk 6 above)
Comment (Yoda-Soda) [reply README.md:105]: yeah like the idea of sharing a repo with your account-a-buddy them pulling your repo down and making changes to their repo. **Q&A get to know your account-a-buddy git challenge.**  Allows people to learn a bit about permissions and the importance of collaboration. Will also avoid people using dropbox for their challenges 😆

File: README.md [hunk 7]
```diff
@@ -1,89 +1,199 @@
...
+### Communication
+
+Install the Slack desktop app. Using the desktop app allows you to make video calls to instructors and other students, which can be handy when you want a 1:1.
+
+- [Slack](https://slack.com/intl/en-nz/) messaging app.
+- [Make calls in Slack](https://slack.com/intl/en-nz/help/articles/216771908-Make-calls-in-Slack)
+
+### Programming tools
+
+- [Node.js](https://nodejs.org/en/) JavaScript library. Make sure to install the LTS version.
```
Comment (lwcooper) [inline README.md:56]:
```suggestion
- [Node.js](https://nodejs.org/en/) JavaScript runtime. Make sure to install the LTS version.

```

File: setup-ssh.sh [hunk 8]
```diff
@@ -1,54 +0,0 @@
-# Instructions from GitHub help pages:
```
Comment (lwcooper) [inline setup-ssh.sh:1]: @Yoda-Soda did you mean to delete this?

File: README.md [hunk 9]
```diff
@@ -1,89 +1,199 @@
...
+Save yourself the hassle of having to learn terminal text editors (like vim) by running these 4 commands:
+
+```bash
+git config --global merge.tool vscode
+git config --global mergetool.vscode.cmd "code --wait $MERGED"
+git config --global diff.tool vscode
+git config --global difftool.vscode.cmd "code --wait --diff $LOCAL $REMOTE"
+```
+
+`Note:` copy, past and run the commands inside your terminal.
```
Comment (lwcooper) [inline README.md:171]:
```suggestion
`Note:` copy, paste and run the commands inside your terminal.

```

**Comment by lwcooper** [comment]
@Yoda-Soda master is going to be all crazy now as I bring changes over from this branch to master, so current students can get the benefits. 

The plan when we are ready to merge this branch will be to not resolve conflicts, and just overwrite them with everything from this branch. 

Therefore, I'm going to leave this branch with conflicts to master for now.

File: README.md (hunk 5 above)
Comment (Yoda-Soda) [reply README.md:113]: Think that would be important. I'll add that

File: README.md [hunk 10]
```diff
@@ -1,80 +1,126 @@
...
+### Web-browser
+
+A modern web browser, preferably [Google Chrome](https://www.google.com/chrome/index.html), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/), or [Brave Browser](https://brave.com/).
+
+- Set your new browser as the default browser for your computer: [Google Chrome](https://support.google.com/chrome/answer/95417?co=GENIE.Platform%3DDesktop&hl=en), [Mozilla Firefox](https://support.mozilla.org/en-US/kb/make-firefox-your-default-browser), or [Brave Browser](https://support.brave.com/hc/en-us/articles/360020406572-How-do-I-set-Brave-to-be-my-Default-Browser-)
+- `Note:` You will need Chrome or the Brave browser to use the Whereby conferencing tool.
+
+### Communication
+
+[Slack](https://slack.com/intl/en-nz/) messaging app.
```
[the remark this answers is not in the document]
Comment (Yoda-Soda) [reply README.md]: Good idea I'll add that.

File: README.md (hunk 9 above)
Comment (Yoda-Soda) [reply README.md:172]: That's so weird I remember fixing that typo 😕

File: setup-ssh.sh (hunk 8 above)
Comment (Yoda-Soda) [reply setup-ssh.sh:1]: No, I didn't mean to delete that.

**Comment by Yoda-Soda** [comment]
> @Yoda-Soda `master` is going to be all crazy now as I manually copy changes from `evaluation`, so current students can get the benefits.
> 
> When we are ready to merge this branch, we **won't** resolve conflicts, instead we will overwrite `master` with `evaluation`
> 
> Therefore, I'm going to leave this branch with conflicts to master for now.

Sounds good this branch can then focus on the next cohort

File: .gitignore [hunk 11]
```diff
@@ -0,0 +1 @@
+~$README.md
```
[the remark this answers is not in the document]
Comment (Yoda-Soda) [reply .gitignore:1]: Not sure I must have added it by accident

File: README.md [hunk 12]
```diff
@@ -1,248 +1,310 @@
...
+- [Enable your home folder in finder](https://vimeo.com/431692277/670a619782)
+- [Enable right mouse click](enable-right-mouse-click-in-macos.md)
+
+## Software
+
+Install the following software:
+
+### Version Control
+
+[Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) by opening up the `terminal` (`⌥option` + `spacebar` and then typing "terminal") and typing in the terminal:
```
Comment (CastleArg) [inline README.md:31]: should be command spacebar

File: README.md [hunk 13]
```diff
@@ -1,248 +1,310 @@
...
+- [Node.js](https://nodejs.org/en/) JavaScript runtime. Make sure to install the LTS version.
+
+### VS Code
+
+- [Visual Studio Code](https://code.visualstudio.com/), the open-source code editor.
+- Customize VS-Code by installing the following extensions:
+  - [ ] [Live Server extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+  - [ ] [Code Runner extension](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner)
+  - [ ] [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+  - [ ] [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare)
```
Comment (CastleArg) [inline README.md:64]: Suggest live share extension pack:
https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare-pack

File: README.md [hunk 14]
```diff
@@ -1,248 +1,310 @@
...
+
+1. Run the following script in your terminal:
+
+   ```bash
+   echo -e $(bash <(curl -s https://raw.githubusercontent.com/Developers-Institute/WDD01-Client-Side-Pre-Course-Setup/master/setup-ssh.sh) YOUR_EMAIL_HERE)
+   ```
+
+   `Note:` Remember to replace "`YOUR_EMAIL_HERE`" with your GitHub email address.
+
+1. The script will automatically copy your public key to your clipboard and open GitHub in your browser. All you need to do is `paste` (⌥option + v) the public key into GitHub and provide it with a title of your choice.
```
Comment (CastleArg) [inline README.md:138]: command v!

File: README.md [hunk 15]
```diff
@@ -1,248 +1,310 @@
...
+If you have been using the internet for some time, there is an unfortunately high chance the password has been compromised.
+
+If your email is compromised, you need to fix that by:
+
+1. Resetting your passwords (unless you already have since the breach date). Any account where you used the same password is potentially compromised.
+1. Stop using the same 3 passwords for everything
+   - How... Sticky notes? Add a number on the end? No 🤦 ! Time to use a password manager 😎 ...
+1. Use a password manager to generate and remember all your passwords
+   - Choose a good master password. Consider a phrase instead of a word. Don't use an old compromised password for this. Don't make it too hard to remember, as forgetting would be a disaster.
+   - Us instructors, use [KeePassXC](https://keepassxc.org/) and [LastPass](https://www.lastpass.com/), and other staff use [Dashlane](https://www.dashlane.com/)
```
Comment (CastleArg) [inline README.md:193]: no comma

File: README.md (hunk 14 above)
Comment (lwcooper) [reply README.md]: Thanks for catching these!

**Review by CastleArg — APPROVED** [review]

**Pull request merged**